dev/sparktestsupport/modules.py

```diff
@@ -15,9 +15,17 @@
 # limitations under the License.
 #
 
+from __future__ import print_function
 from functools import total_ordering
 import itertools
 import re
+import os
+
+if os.environ.get("AMPLAB_JENKINS"):
```

Comment:
@wangyum, this will shows the info every time this modules is imported. why did we do this here?

Reply:
okay. it's a temp fix so I'm fine. I will make a followup to handle https://github.com/apache/spark/pull/24639/files

Reply:
Skip the `hive-thriftserver` module when running the hadoop-3.2 test. Will remove it in another PR: https://github.com/apache/spark/pull/24628/files

Reply:
Thank you @HyukjinKwon